README.md

```diff
@@ -229,9 +229,9 @@ In order to use the external environment functionality you must set the
 environments in ```config/azure_environments.yml ```like so:  
 ```
 qa:
-    webapp: <webapp>
-    rgroup: <rgroup>
-    subscription: <subscription>
+  webapp: <webapp>
```

Comment:
is it missing rollover?

`mcb Command` do we need to add a bit concerning rollover for the future?

Reply:
I think it's temporary and not worth adding, tbh.

Reply:
until next year then